src/components/NewCampaignForm.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <form @submit.prevent='addCampaign'>
+    <input v-model='title' placeholder='Campaign Title' />
+    <input v-model='minContribution' placeholder='Minimun Wei Contribution' />
+    <textarea v-model='description' placeholder='Describe your Campaign...'></textarea>
+    <button type='submit' > Create Campaign </button>
+  </form>
+</template>
+
+<script>
+export default {
+  name: 'NewCampaignForm',
+  data() {
+    return {
+      title: '',
+      description: '',
+      minContribution: null
+    }
+  },
+  methods: {
+    addCampaign() {
+      const newCampaign = {
+        title: this.title,
+        description: this.description,
+        minContribution: this.minContribution
+      }
+      this.$store.commit('ADD_CAMPAIGN', newCampaign)
+    }
```

Comment:
Creates campaign object and 'commits' it to the store. Use of store mutations.

Reply:
Looks great, I am just about ready to establish our connection to the blockchain. Maybe we can set this up this afternoon.